app/components/Home.js

```diff
@@ -174,14 +175,23 @@ export default class Home extends React.Component {
     }, 2000);
   };
 
+  onSearch = event => {
+    this.setState({ searchTerm: event.target.value })
+  }
+
   render() {
     const editEntry = this.state.editEntry || {};
+
+    this.state.entries.map(entry => {
+      entry.filtered = Object.keys(entry).some(key => String(entry[key]).toLowerCase().includes(this.state.searchTerm.toLowerCase())) || !this.state.searchTerm
```

Comment:
Can probably use Array.filter();

Something like:

```
const entries = entries.filter(...)
setState({ entries: entries )}
```

Reply:
I tried using `filter`, but then it would get too complex without refactoring edit and remove logic (due to the IDs problem mentioned on Slack).